react/features/large-video/components/LargeVideo.web.js

```diff
@@ -37,6 +37,13 @@ export default class LargeVideo extends Component {
                         src = '' />
                 </div>
                 <span id = 'remoteConnectionMessage' />
+                <div className = 'video_blurred_container'>
```

Comment:
I think this implementation was needed for the desired effect. When a portrait-oriented video is displayed, it's not zoomed in and instead will have empty space on its sides. To prevent the background from being black, something was added behind the video, and that something was the zoomed in version of the video with movement and all but blurred.